drivers/crc/crc16_sw.c

```diff
@@ -34,3 +34,21 @@ u16_t crc16(const u8_t *src, size_t len, u16_t polynomial,
 
 	return crc;
 }
+
+u16_t crc16_ccitt(const u8_t *src, size_t len)
+{
+	u16_t crc = 0x1d0f;
+	size_t i;
+
+	for (i = 0; i < len; i++) {
+		/*
+		 * See https://en.wikipedia.org/wiki/
```

Comment:
remove that comment, you already clearly pint pointed the source in your commit message.